lib/Elastica/Exception/Connection/GuzzleException.php

```diff
@@ -21,9 +21,7 @@ class GuzzleException extends ConnectionException
     protected $_guzzleException;
 
     /**
-     * @param TransferException $guzzleException
-     * @param Request           $request
-     * @param Response|null     $response
+     * @param Request $request
```

Comment:
Not sure why only `$request` is kept here :confused:

Reply:
For the same reason probably I explained below